docassemble/StudentEvaluations/data/questions/prs_complaint_interview.yml

```diff
@@ -230,8 +232,10 @@ question: |
 subquestion: |
   This should be the name of the parent or legal guardian of ${ student }.
 fields:
-  - code: |
-      parent.name_fields()
+  - First name: parent.name.first
```

Comment:
See better way to fix suffix

https://suffolklitlab.org/docassemble-AssemblyLine-documentation/docs/question_library/names#optional-omit-suffix